utils/jmxfiles.py

```diff
@@ -0,0 +1,109 @@
+# std
+import logging
+import os
+import tempfile
+import time
+
+# 3rd party
+import yaml
+
+# datadog
+from config import _windows_commondata_path
+from util import yDumper
+from utils.pidfile import PidFile
+from utils.platform import Platform
+
+
+log = logging.getLogger(__name__)
+
+
+class JMXFiles(object):
```

Comment:
The only point that could be taken here, is that bundling the methods in a class is easier for imports and there is much less risk to override an existing method from another module.